examples/injection-of-env-entry/README_pt.adoc

```diff
@@ -0,0 +1,264 @@
+:index-group: Environment Entries
+:jbake-type: page
+:jbake-status: status=published
+= Usando EnvEntries
+
+A anotação `@Resource` pode ser usada para injetar várias coisas, incluindo fontes de dados, tópicos, filas etc. A maioria delas são objetos fornecidos por contêiner.
+
+É possível, no entanto, fornecer seus próprios valores para serem injetados através de um
+`<env-entry>` no seu descritor de implementação `ejb-jar.xml` ou `web.xml`.
+Os tipos `<env-entry>` suportados pelo Java EE 6 são limitados ao seguinte:
+
+* java.lang.String
+* java.lang.Integer
+* java.lang.Short
+* java.lang.Float
+* java.lang.Double
+* java.lang.Byte
+* java.lang.Character
+* java.lang.Boolean
+* java.lang.Class
+* java.lang.Enum (any enum)
+
+Consulte também o link:../custom-injection[Injeção personalizada] exemplo para um
+recurso TomEE e OpenEJB permitem que você use mais do que apenas os tipos acima, além de declarar itens `<env-entry>` com um arquivo de propriedades simples.
+
+== Usando @Resource para propriedades básicas
+
+O uso da anotação `@Resource` não se limita aos setters.
+Por exemplo, esta anotação poderia ter sido usada no correspondente
+_campo_ assim:
+
+....
+@Resource
+private int maxLineItems;
+....
+
+Um exemplo mais completo pode ser assim:
+
+....
+package org.superbiz.injection.enventry;
+
+import javax.annotation.Resource;
+import javax.ejb.Singleton;
+import java.util.Date;
+
+@Singleton
+public class Configuration {
+
+    @Resource
+    private String color;
+
+    @Resource
+    private Shape shape;
+
+    @Resource
+    private Class strategy;
+
+    @Resource(name = "date")
+    private long date;
+
+    public String getColor() {
+        return color;
+    }
+
+    public Shape getShape() {
+        return shape;
+    }
+
+    public Class getStrategy() {
+        return strategy;
+    }
+
+    public Date getDate() {
+        return new Date(date);
+    }
+}
+....
+
+Aqui temos um bean `@Singleton` chamado `Confuration` que possui as seguintes propriedades (itens `<env-entry> `)
```

Comment:
Dani, i believe that "Confuration" is "Configuration"

Reply:
done thanks.